### PR TITLE
Add custom error message for better traceability for multiple asserts

### PR DESCRIPTION
**Pull request by dimitrovmaksim**
Signed-off-by: Maksim Dimitrov <dimitrov.maksim@gmail.com>

**Description**:
Adds custom messages in cases with multiple expect/asserts for better traceability

**Related issue(s)**:

Fixes https://github.com/hashgraph/hedera-json-rpc-relay/issues/774

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by georgi-l95 — APPROVED** [review]
LG

File: packages/server/tests/helpers/assertions.ts [hunk 1]
```diff
@@ -37,14 +37,14 @@ export default class Assertions {
...
-        expect(num).to.not.be.null;
+        expect(shard, 'Id shard should not be null').to.not.be.null;
+        expect(realm, 'Id realm should not be null').to.not.be.null;
+        expect(num, 'Id num should not be null').to.not.be.null;
     };
 
     static unsupportedResponse = (resp: any) => {
-        expect(resp.error.code).to.eq(-32601);
-        expect(resp.error.message.endsWith('Unsupported JSON-RPC method')).to.be.true;
+        expect(resp.error.code, 'Unsuported response.error.code should equal -32601').to.eq(-32601);
```
Comment (lukelee-sl) [inline packages/server/tests/helpers/assertions.ts:46]: nit: typo Unsuported->Unsupported

File: packages/server/tests/helpers/assertions.ts [hunk 2]
```diff
@@ -69,33 +69,33 @@ export default class Assertions {
...
-        expect(relayResponse.nonce).to.be.equal(Assertions.zeroHex8Byte);
-        expect(relayResponse.receiptsRoot).to.be.equal(Assertions.zeroHex32Byte);
-        expect(relayResponse.sha3Uncles).to.be.equal(Assertions.emptyArrayHex);
-        expect(relayResponse.stateRoot).to.be.equal(Assertions.zeroHex32Byte);
-        expect(relayResponse.totalDifficulty).to.be.equal(ethers.utils.hexValue(0));
-        expect(relayResponse.uncles).to.be.exist;
-        expect(relayResponse.uncles.length).to.eq(0);
-        expect(relayResponse.logsBloom).to.eq(Assertions.emptyBloom);
-        expect(relayResponse.gasLimit).to.equal(ethers.utils.hexValue(Assertions.maxBlockGasLimit));
+        expect(relayResponse.difficulty, "Assert block: 'dificulty' shuld equal zero in hex").to.be.equal(ethers.utils.hexValue(0));
```
Comment (lukelee-sl) [inline packages/server/tests/helpers/assertions.ts:72]: nit: more typos here and below

Comment (dimitrovmaksim) [reply packages/server/tests/helpers/assertions.ts]: 🤦🏻

**Review by lukelee-sl — APPROVED** [review]
LG

**Review by georgi-l95 — APPROVED** [review]

**Pull request merged**